graal/backends/core/cocom.py

```diff
@@ -149,7 +170,7 @@ def _analyze(self, commit):
         """
         analysis = []
 
-        if self.analyzer_kind == LIZARD_FILE:
+        if self.analyzer_kind == LIZARD_FILE or self.analyzer_kind == SCC_FILE:
```

Comment:
maybe this could be replaced with `if self.analyzer_kind in [LIZARD_FILE, SCC_FILE]`